analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MethodParameterUnused.CSharp8.cs

```diff
@@ -66,6 +68,13 @@ public void Method()
                 var result = a + c;
             }
         }
+
+        // See: https://github.com/SonarSource/sonar-dotnet/issues/3803
+        private void AddValue(uint id1, uint id2, string value) // Noncompliant - FP: id1 and id2 are used inside the tuple
```

Comment:
CI failed, we'll need one more `//Noncompliant@-1` marker

Reply:
Indeed, there are two of them 😄